docs/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _readme:
+
+postfix-formula
+===============
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/postfix-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/postfix-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+A SaltStack formula that is empty. It has dummy content to help with a quick
+start on a new formula and it serves as a style guide.
+
+.. contents:: **Table of Contents**
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see :ref:`How to contribute <CONTRIBUTING>` for more details.
+
+Available states
+----------------
+
+.. contents::
+   :local:
+
+``postfix``
+^^^^^^^^^^^
+
+Installs and starts postfix SMTP server
+
+``postfix.config``
+^^^^^^^^^^^^^^^^^^
+
+Manages postfix main.cf and optionally the master.cf configuration file
+
+``postfix.policyd-spf``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Installs and configures policyd-spf
+
+``postfix.postgrey``
+^^^^^^^^^^^^^^^^^^^^
+
+Installs and starts Postgrey service
+
+``postfix.mysql``
+^^^^^^^^^^^^^^^^^
+
+Installs postfix mysql package ( Debian only)
+
+``postfix.pcre``
+^^^^^^^^^^^^^^^^
+
+Installs postfix pcre package ( Debian only)
+
+``postfix.postsrsd``
+^^^^^^^^^^^^^^^^^^^^
+
+Installs postfix postsrsd package
+
+
+Testing
+-------
+
+Linux testing is done with ``kitchen-salt``.
+
+Requirements
+^^^^^^^^^^^^
+
+* Ruby
+* Docker
+
+.. code-block:: bash
+
+   $ gem install bundler
+   $ bundle install
+   $ bin/kitchen test [platform]
+
+Where ``[platform]`` is the platform name defined in ``kitchen.yml``,
+e.g. ``debian-9-2019-2-py3``.
+
+``bin/kitchen converge``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Creates the docker instance and runs the main(s) state(s), ready for testing.
```

Comment:
I usually check `kitchen.yml` to see which states are being run.  But something like this could be a better solution in the long-term, by fixing it in the `template-formula`.  Not really `main(s) state(s)`, though.  Perhaps `main state(s)` is good enough.

Reply:
yes, `main state(s)` is more correct